### ISSUE TITLE
Build is not passing lint

### ISSUE THREAD

**Issue by coopy** [FormidableLabs/component-playground#52]
Repro:

`npm run check`

See all the lint fail.

```
src/components/editor.jsx
  12:24  error  'theme' is missing in props validation      react/prop-types
  13:27  error  'readOnly' is missing in props validation   react/prop-types
  19:19  error  'readOnly' is missing in props validation   react/prop-types
  19:42  error  'external' is missing in props validation   react/prop-types
  20:38  error  'codeText' is missing in props validation   react/prop-types
  25:20  error  'readOnly' is missing in props validation   react/prop-types
  25:43  error  'onChange' is missing in props validation   react/prop-types
  26:17  error  'onChange' is missing in props validation   react/prop-types
  31:4   error  Unexpected var, use let or const instead    no-var
  31:65  error  'codeText' is missing in props validation   react/prop-types
  34:29  error  'style' is missing in props validation      react/prop-types
  34:58  error  'className' is missing in props validation  react/prop-types

src/components/es6-preview.jsx
    2:0   error  Unnecessary "use strict" directive              strict
   51:8   error  `pairs` is never modified, use `const` instead  prefer-const
   54:13  error  `key` is never modified, use `const` instead    prefer-const
   95:4   error  "clearTimeout" is not defined                   no-undef
  115:4   error  "clearTimeout" is not defined                   no-undef
  116:21  error  "setTimeout" is not defined                     no-undef
  120:4   error  Unexpected var, use let or const instead        no-var
  125:6   error  Unexpected console statement                    no-console
  125:6   error  "console" is not defined                        no-undef
  129:6   error  Unexpected var, use let or const instead        no-var
  130:11  error  Unexpected var, use let or const instead        no-var
  136:6   error  Unexpected var, use let or const instead        no-var
  137:6   error  Unexpected var, use let or const instead        no-var
  152:17  error  eval can be harmful                             no-eval
  152:58  error  There should be no spaces inside this paren     space-in-parens

src/components/playground.jsx
   2:0   error  Unnecessary "use strict" directive              strict
  68:46  error  "console" is not defined                        no-undef
  69:10  error  "console" is not defined                        no-undef
  70:0   error  Line 70 exceeds the maximum line length of 100  max-len

src/components/preview.jsx
   1:0   error  Unnecessary "use strict" directive              strict
  23:26  error  Strings must use doublequote                    quotes
  24:7   error  Missing semicolon                               semi
  32:6   error  "clearTimeout" is not defined                   no-undef
  39:21  error  'noRender' is missing in props validation       react/prop-types
  49:67  error  'context' is missing in props validation        react/prop-types
  50:80  error  'context' is missing in props validation        react/prop-types
  58:54  error  There should be no spaces inside this paren     space-in-parens
  69:6   error  "clearTimeout" is not defined                   no-undef
  70:23  error  "setTimeout" is not defined                     no-undef
  74:6   error  Unexpected var, use let or const instead        no-var
  78:8   error  Unexpected var, use let or const instead        no-var
  80:13  error  Unexpected var, use let or const instead        no-var
  88:8   error  Unexpected var, use let or const instead        no-var
  89:23  error  'noRender' is missing in props validation       react/prop-types
  90:10  error  Unexpected var, use let or const instead        no-var
  91:12  error  eval can be harmful                             no-eval
  93:0   error  Line 93 exceeds the maximum line length of 100  max-len
  99:10  error  eval can be harmful                             no-eval

demo/app.jsx
   2:0   error  Unnecessary "use strict" directive        strict
   4:0   error  Unexpected var, use let or const instead  no-var
   5:0   error  Unexpected var, use let or const instead  no-var
   6:0   error  Unexpected var, use let or const instead  no-var
  12:0   error  Unexpected var, use let or const instead  no-var
  13:0   error  Unexpected var, use let or const instead  no-var
  15:0   error  Unexpected var, use let or const instead  no-var
  16:0   error  Unexpected var, use let or const instead  no-var
  18:0   error  Unexpected var, use let or const instead  no-var
  20:0   error  Unexpected var, use let or const instead  no-var
  28:18  error  Expected property shorthand               object-shorthand
  28:32  error  Expected property shorthand               object-shorthand
  34:18  error  Expected property shorthand               object-shorthand
  34:32  error  Expected property shorthand               object-shorthand
  41:18  error  Expected property shorthand               object-shorthand
  41:32  error  Expected property shorthand               object-shorthand
  49:18  error  Expected property shorthand               object-shorthand
  49:32  error  Expected property shorthand               object-shorthand
  61:18  error  Expected property shorthand               object-shorthand
  61:32  error  Expected property shorthand               object-shorthand
  68:18  error  Expected property shorthand               object-shorthand
  68:32  error  Expected property shorthand               object-shorthand
  74:26  error  "document" is not defined                 no-undef

✖ 73 problems (73 errors, 0 warnings)
```

**Comment by kenwheeler** [comment]
Want me to clean this up @coopy ?

**Comment by coopy** [comment]
That would be great; if so, please also change back the `preversion` script
to run `npm run check`.
lör 27 feb. 2016 kl. 11:33 skrev Ken Wheeler <notifications@github.com>:

> Want me to clean this up @coopy <https://github.com/coopy> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FormidableLabs/component-playground/issues/52#issuecomment-189709307>
> .
>
-- 
*Per Nilsson*
Software Architect *Formidable*
per.nilsson@formidable.com
(512) 550-7241

**Comment by kenwheeler** [comment]
On it

**Comment by NogsMPLS** [comment]
Issue should be closed now with #55
